Sources/HaCWebsiteLib/Utils/DateUtils.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+enum DateUtils {
+  private static let locale = Locale(identifier: "en_GB")
+
+  static let individualDayFormatter: DateFormatter = {
+    let formatter = DateFormatter()
+    formatter.dateStyle = .medium
+    formatter.timeStyle = .none
+    formatter.locale = locale
+
+    return formatter
+  }()
+}
+
+extension Date {
+  static func from(year: Int, month: Int, day: Int, hour: Int, minute: Int, timezone: TimeZone? = nil) -> Date! {
```

Comment:
What's the reasoning for using an implicitly unwrapped optional here?

Reply:
Reasoning was that it'd be nice if we don't have to propagate optionals all over here. Since we're hard coding dates for now, we don't want the site to successfully run with a bad date hanging around so its appropriate to crash. Maybe we should guarantee that more explicitky

Reply:
Yeah should we crash at this stage instead of returning a time-bomb?